geoips/commandline/geoips_config.py

```diff
@@ -48,10 +49,10 @@ def __call__(self, args):
         """
         test_dataset_name = args.test_dataset_name
         test_dataset_url = test_dataset_dict[test_dataset_name]
-        if test_dataset_name in listdir(self.geoips_testdata_dir):
+        if any([test_dataset_name in fol for fol in listdir(self.geoips_testdata_dir)]):
             print(
                 f"Test dataset '{test_dataset_name}' already exists under "
-                f"'{join(self.geoips_testdata_dir, test_dataset_name)}'. See that "
+                f"'{join(self.geoips_testdata_dir, test_dataset_name)}*/'. See that "
```

Comment:
This does not decompress the test datasets into test_data_<sensor>_<version> right?  The final directories are $GEOIPS_TESTDATA_DIR/test_data_<sensor> correct?

Reply:
Correct!